minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -838,7 +838,7 @@ public synchronized void stop() throws IOException, InterruptedException {
     // and not wait for the old locks to be cleaned up.
     try {
       new ZooZap().zap(getServerContext().getSiteConfiguration(), "-manager",
-          "-compaction-coordinators", "-tservers", "-compactors", "-sservers");
+          "-compaction-coordinators", "-tservers", "-compactors", "-sservers", "--gc", "--monitor");
```

Comment:
mini does not start the monitor. Can leave as-is if it won't cause an error.

Reply:
Removed that option in 5c4b810 because its misleading.  Could make someone think that mini does start the monitor.